e2e/enable-receipts-v2-feature.toml

```diff
@@ -9,12 +9,12 @@
 [[TestCases]]
    RunCmd = "{{ $.LoomPath }} chain-cfg list-features"
    Condition = "excludes"
-   Expected = ['receipts:v2']
+   Excluded = ['receipts:v2']
 
 [[TestCases]]
   RunCmd = "/bin/ls ./0"
-  Condition = "excludes"
-  Expected = ['receipts_db']
+  Condition = "contains"
+  Excluded = ['receipts_db']
```

Comment:
`contains` & `Expected`, or `excludes` & `Excluded`... which one should this be?

Reply:
Should be `contains` & `Expected`.